src/test/groovy/graphql/execution/incremental/DeferExecutionSupportIntegrationTest.groovy

```diff
@@ -162,19 +162,25 @@ class DeferExecutionSupportIntegrationTest extends Specification {
                         .dataFetcher("hello", resolve("world"))
                         .dataFetcher("item", resolveItem())
                 )
-                .type(newTypeWiring("Post").dataFetcher("summary", resolve("A summary", 10)))
-                .type(newTypeWiring("Post").dataFetcher("text", resolve("The full text", 100)))
-                .type(newTypeWiring("Post").dataFetcher("wordCount", resolve(45999, 10, true)))
-                .type(newTypeWiring("Post").dataFetcher("latestComment", resolve([title: "Comment title"], 10)))
-                .type(newTypeWiring("Post").dataFetcher("dataFetcherError", resolveWithException()))
-                .type(newTypeWiring("Post").dataFetcher("dataAndError", resolveWithDataAndError("data")))
-                .type(newTypeWiring("Post").dataFetcher("coercionError", resolve("Not a number", 10)))
-                .type(newTypeWiring("Post").dataFetcher("typeMismatchError", resolve([a: "A Map instead of a List"], 10)))
-                .type(newTypeWiring("Post").dataFetcher("nonNullableError", resolve(null)))
-                .type(newTypeWiring("Page").dataFetcher("summary", resolve("A page summary", 10)))
-                .type(newTypeWiring("Page").dataFetcher("text", resolve("The page full text", 100)))
-                .type(newTypeWiring("Comment").dataFetcher("content", resolve("Full content", 100)))
-                .type(newTypeWiring("Comment").dataFetcher("author", resolve([name: "Author name"], 10)))
+                .type(newTypeWiring("Post")
```

Comment:
Some test updates, extra datafetcher registrations should happen inside the same newTypeWiring builder